agent/ecscni/types.go

```diff
@@ -156,4 +184,20 @@ type Config struct {
 	AdditionalLocalRoutes []cnitypes.IPNet
 	// SubnetGatewayIPV4Address is the address to the subnet gate for the eni
 	SubnetGatewayIPV4Address string
+	// App Mesh CNI enabled
```

Comment:
[multiple occurrences] please do read https://golang.org/doc/effective_go.html#commentary for how to comment in golang. This should read something like: `// AppMeshCNIEnabled is ...`

Reply:
sounds good, updated